### PR TITLE
iOS support for timers and notifications and other goodies

### PR DESCRIPTION
**Pull request by j-koreth**

### CODE REVIEW

**Comment by j-koreth** [comment]
Other things include updating the gradle version and changed how timers were handled.

File: lib/src/models/timer.dart [hunk 1]
```diff
@@ -41,7 +41,10 @@ class Timer {
     this.duration, [
     tz.TZDateTime done,
   ]) {
+    print(tz.local);
```
Comment (Teifun2) [inline lib/src/models/timer.dart:44]: Debugging statements

File: lib/src/widget/animated_time_progress_bar.dart [hunk 2]
```diff
@@ -35,7 +35,11 @@ class _AnimatedTimeProgressBarState extends State<AnimatedTimeProgressBar>
       duration: _timer.remaining(),
       vsync: this,
     );
-    this._controller.forward();
+
+    this._controller.forward().whenCompleteOrCancel(() {
+      print("Completed Animation Controller???");
```
Comment (Teifun2) [inline lib/src/widget/animated_time_progress_bar.dart:40]: Debugging Statement

**Review by Teifun2 — CHANGES_REQUESTED** [review]
Thank you very much for this PR. This looks very promissing. I also sotted the small changes to the notification and i like them a lot!

I have one question. Do we need "Round" Icon assets for IOS? The flutter default Icon seemed to have transparent round corners, is this needed for good looks or will IOS make it round on its own?

**Comment by j-koreth** [comment]
No I don't think Round icon assets are needed for iOS on my devices IOS rounds them out by default. 
I added a new commit with the debugging statements removed. Also do we need the timer_builder package still?

**Comment by Teifun2** [comment]
> Also do we need the timer_builder package still?

You are correct it is no longer needed!

**Comment by j-koreth** [comment]
Done 
sorry for the delay

**Comment by Teifun2** [comment]
> Done sorry for the delay

No hurry :)
Thanks a lot for the addition!

**Review by Teifun2 — APPROVED** [review]

**Pull request merged**